domains/client/domain-operator/src/fraud_proof.rs

```diff
@@ -434,6 +434,7 @@ where
             &*self.backend,
             extrinsics.into(),
             None,
+            false,
```

Comment:
Why `false` here and not checking for Gemini 3h?

Reply:
Here, the inherent extrinsic is contained in the `extrinsics` param, which is getting from the block body and ordered, so we are passing `maybe_inherent_data` as `None` and the value of `is_gemini_3h` is not used when `maybe_inherent_data` is `None`.

Reply:
It really wasn't super obvious why `is_gemini_3h: false` is okay even on Gemini 3h, maybe add a comment here about this?